sdk/src/main/scala/io/horizen/utxo/SidechainApp.scala

```diff
@@ -100,7 +95,8 @@ class SidechainApp @Inject()
   override val swaggerConfig: String = Source.fromResource("utxo/api/sidechainApi.yaml")(Codec.UTF8).getLines.mkString("\n")
 
   override protected lazy val sidechainTransactionsCompanion: SidechainTransactionsCompanion = SidechainTransactionsCompanion(customTransactionSerializers, circuitType)
-  protected lazy val sidechainBoxesCompanion: SidechainBoxesCompanion = SidechainBoxesCompanion(customBoxSerializers, sc2scConfigurator.canSendMessages)
+  private lazy val sc2ScFork = Sc2ScFork.get(TimeToEpochUtils.timeStampToEpochNumber(params, timeProvider.time()))
```

Comment:
Is it safe using the timeProvider for getting the fork at the startup? It might not be active yet